BaSS.Rproj

```diff
@@ -0,0 +1,21 @@
+Version: 1.0
```

Comment:
Good point, actually, I did not even notice this since I don't edit this file.  The Description file matches GitHub.  Where can you set this without editing it in a text editor?

Reply:
Not sure I've seen a good way to automate this ... Looks like [{tidyverse} doesn't sync versions](https://github.com/tidyverse/tidyverse/blob/master/tidyverse.Rproj), so probably ok for us to let this one slide as well.